demos/python_demos/multi_camera_multi_person_tracking/requirements.txt

```diff
@@ -1,3 +1,11 @@
-opencv-python
-numpy
-scipy
+Cython>=0.29.12
```

Comment:
Cython doesn't seem to be used anywhere.

Reply:
You are right. Removed.